.storybook/stories/checkbox-toggle/checkbox-toggle.stories.ts

```diff
@@ -56,22 +41,83 @@ const defaultParameters: Parameters = {
   },
 };
 
-setupStorybook(ClrCheckboxModule, defaultStory, defaultParameters, generateVariants());
+const CheckBoxToggleTemplate: Story = args => {
+  const containerSelector = args.type === CheckboxType.Checkbox ? 'clr-checkbox-container' : 'clr-toggle-container';
+  const wrapperSelector = args.type === CheckboxType.Checkbox ? 'clr-checkbox-wrapper' : 'clr-toggle-wrapper';
+  const directive = args.type === CheckboxType.Checkbox ? 'clrCheckbox' : 'clrToggle';
 
-function generateVariants() {
-  const variants: Parameters[] = [];
+  return {
+    template: `
+      <${containerSelector}> <!-- The container is required in this story so that the disabled state works correctly. -->
+        <${wrapperSelector}>
+          <input type="checkbox" ${directive} [ngModel]="checked" [disabled]="disabled" />
+          <label>{{label}}</label>
+        </${wrapperSelector}>
+      </${containerSelector}>
+    `,
+    props: args,
+  };
+};
 
-  for (const type of [CheckboxType.Checkbox, CheckboxType.Toggle]) {
-    for (const disabled of [false, true]) {
-      for (const checked of [false, true]) {
-        variants.push({
-          type,
-          disabled,
-          checked,
-        });
-      }
-    }
-  }
+const CheckBoxTemplate: Story = args => {
+  const containerSelector = args.type === CheckboxType.Checkbox ? 'clr-checkbox-container' : 'clr-toggle-container';
+  const wrapperSelector = args.type === CheckboxType.Checkbox ? 'clr-checkbox-wrapper' : 'clr-toggle-wrapper';
+  const directive = args.type === CheckboxType.Checkbox ? 'clrCheckbox' : 'clrToggle';
 
-  return variants;
-}
+  return {
+    template: `
+    <h6>Enabled</h6>
```

Comment:
I am concerned with using `h6` like this. This can cause a11y violations in our stories. I think we should use the appropriate `cds-text` option for styling instead of the tag name.

Reply:
Replaced the `<h6>`   with  `<span cds-text="subsection">Inline</span>`